select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java

```diff
@@ -42,10 +44,13 @@
 		public static String ELEMENT = "element";
 	}
 
+	// language code (e.g. "en", "de", "fr", ...)
+	private String language;
+
 	private Integer minimumInputLength, minimumResultsForSearch;
 	private Integer maximumSelectionLength;
-	private Object placeholder;
-	private boolean allowClear;
+	private Object placeholder; // ok
```

Comment:
Oops, shouldn't have made it into the commit, was about to check which settings/options are still valid with select2 4.x. Will remove this in a minute ...